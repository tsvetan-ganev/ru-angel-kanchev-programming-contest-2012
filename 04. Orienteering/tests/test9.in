17 2
0301Gospodinova w 10:24:08 10:56:55
1501Raykova w 10:23:29 10:53:19
1701Atanasova w 10:58:47 11:24:48
1402Nikolova w 08:22:28 09:04:02
0501Kostova w 10:00:59 10:49:59
0401Marinova w 08:29:43 08:55:22
1102Kostadinova w 09:53:31 10:21:12
0201Hristova w 08:06:04 08:32:05
0602Drashkova w 08:55:11 09:47:32
0102Stoycheva w 08:12:39 08:53:25
1202Petkova w 08:11:05 08:41:45
0801Stoyanova w 09:53:45 10:48:14
1302Dancheva w 10:34:38 11:08:31
1002Pencheva w 09:56:46 10:31:00
0701Donevska w 08:34:43 09:20:47
1601Stoyanova w 10:16:28 10:55:54
0902Izvorova w 08:27:35 08:52:45