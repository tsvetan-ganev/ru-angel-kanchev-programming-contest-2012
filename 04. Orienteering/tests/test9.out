
09 04 02 17 
01 